search-resources-api/src/main/java/no/unit/nva/search2/SearchTicketAuthHandler.java

```diff
@@ -58,6 +60,8 @@ protected List<MediaType> listSupportedMediaTypes() {
 
     @Override
     protected void validateRequest(Void unused, RequestInfo requestInfo, Context context) throws ApiGatewayException {
-        //Do nothing
+        if (Objects.isNull(requestInfo.getUserName())) {
+            throw new UnauthorizedException();
+        }
```

Comment:
Nice

Reply:
I do check for access in QueryBuilder as well, but it should be faster to check before invoking the builder.